src/geometric_utils.ts

```diff
@@ -313,13 +356,16 @@ export class GeometricUtils {
         if (poly.length > 2) {
             try {
                 ret = poly[0][0] === poly.at(-1)[0] && poly[0][1] === poly.at(-1)[1];
-            } catch (e) {}
+            } catch { /* empty */ }
```

Comment:
This is from the [no-empty](https://eslint.org/docs/latest/rules/no-empty) rule. The ESLint vscode plugin had a fix of adding a comment like this.

I'd be fine disabling this rule, but perhaps it serves a purpose in that you know a block is intentionally empty.